topics/task-worker.yaml

```diff
@@ -8,10 +8,9 @@ services:
     - getsentry/sentry
 schemas:
   - version: 1
-    compatibility_mode: none
-    # TODO(mark) protobuf support
-    type: json
-    resource: taskworker.v1.schema.json
+    compatibility_mode: backward
```

Comment:
how does this compatibility mode actually affect functionality?

Reply:
I had interpreted this as indicating that the messages in the topic were backwards compatible. However, we have no topics with this mode right now, which makes me think it may not be complete.